packages/web-client/app/components/common/degraded-service-banner/index.css

```diff
@@ -15,11 +15,35 @@
   color: var(--degraded-service-banner-text-color);
 }
 
+.degraded-service-banner__container {
+  max-width: 80%;
```

Comment:
Minor since design updates may be incoming: `max-width: calc(var(--icon-space) + 40ch);`? This would make it work across most screen sizes without the need for a media query, and directly address the intention of restricting text length for readability purposes.

(requires setting variable(s) that represents length - icon width + margin)